library/multipicker/src/main/java/im/vector/lib/multipicker/Picker.kt

```diff
@@ -91,6 +94,7 @@ abstract class Picker<T> {
         } else if (dataUri != null) {
             selectedUriList.add(dataUri)
         } else {
+            @Suppress("DEPRECATION")
```

Comment:
why did you add the deprecation here?

Reply:
`get` is deprecated, but since we check for multiple types, I cannot use the new API.